static/app/components/nav/primary/components.tsx

```diff
@@ -123,62 +129,54 @@ export function SidebarLink({
   );
 }
 
-const SidebarItemWrapper = styled('li')<{isMobile: boolean}>`
-  svg {
-    --size: 14px;
-    width: var(--size);
-    height: var(--size);
-
-    ${p =>
-      !p.isMobile &&
-      css`
-        --size: 16px;
-      `}
-  }
-  > a,
-  button {
-    display: flex;
-    flex-direction: row;
-    gap: ${space(1.5)};
-    align-items: center;
-    padding: ${space(1.5)} ${space(3)};
+const baseNavItemStyles = (p: {isMobile: boolean; theme: Theme}) => css`
```

Comment:
Extracted these styles out of SidebarItemWrapper and applying it individually to avoid the specificity of `>`